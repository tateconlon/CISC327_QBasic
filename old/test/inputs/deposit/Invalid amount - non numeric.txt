login
machine
deposit
1234567
eleventeen
logout