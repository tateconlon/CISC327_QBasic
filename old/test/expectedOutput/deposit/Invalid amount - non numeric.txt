login as machine successful
eleventeen is not a valid number
Transaction summary written and system logged out successfully